CRM/Utils/Check/Component/Source.php

```diff
@@ -76,10 +76,15 @@ public function findOrphanedFiles() {
         ]);
       }
       if (file_exists($path)) {
-        $orphans[] = [
-          'name' => $file,
-          'path' => $path,
-        ];
+        if ((strpos($file, '[civicrm.vendor]') !== FALSE || strpos($file, '[ivicrm.packages]') !== FALSE) && strpos($path, 'civicrm/') === FALSE) {
```

Comment:
Haven't really looked yet just noting typo: `ivicrm.packages`.

Reply:
Brain is slowing down for the day but I think the last clause is going to still hit packages either way. I mean:

drupal 7: /path/to/drupal/sites/all/modules/civicrm/packages
drupal 8: /path/to/drupal/vendor/civicrm/civicrm-packages

They both have `civicrm/` in them.

And on windows depending on the setup it might be `civicrm\`.

Given all except one of the files being checked is under either packages or vendor, would it just make sense to skip the whole status check if CIVICRM_UF==Drupal8?

Reply:
The problem is as @MikeyMJCO pointed out at one point is that some people may have created their drupal project where by the vendor directory is within the webroot which thinking about it more this doesn't actually solve but hmm, Not sure if Mikey has any creative thoughts to solve this one